CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Validators for `Campaign` attributes
 _ `_optional` subpackage for managing optional dependencies
 - Acquisition function for active learning: `qNIPV`
+- Acquisition function: `qKG` (knowledge gradient)
```

Comment:
Now that I see this, perhaps you  can also add what `qNIPV` stands for?

Reply:
will do as part of #273 because it modifies that line by adding an entry